### DISCUSSION TITLE
OpenMetadata and its place amongst Data Mesh concepts

### DISCUSSION THREAD

**Discussion by juliarvalenti** [open-metadata/OpenMetadata#676] [General]
Hello!

I have a few questions about the roadmap and planned feature set for the future of OpenMetadata.

Our team has done a lot of research on the topic of [Data Mesh](https://martinfowler.com/articles/data-mesh-principles.html).  OpenMetadata provides a good number of features already for a data mesh including improving things like discoverability and interoperability.  My questions today are mainly centered around where you see the future of OpenMetadata developing specifically in regards to the Data Mesh concept.

1. Is there a plan for OpenMetadata to support any features related to Data Governance or Residency?  Enforcing compliance with various regulations (e.g. GDPR, HIPAA, CCPA) is a big topic these days and I'm just curious if the team is experimenting with some ways on assimilating compliance into the Metadata model.

2. Are there any plans to incorporate features regarding data quality and ensuring the data quality within the platform is held to a certain standard?  I saw in some mockups [posted in a previous discussion](https://github.com/open-metadata/OpenMetadata/discussions/628) that there is a field for null ratios on columns of a table.  This is a great step for data quality measurements, but are there any further plans to go down this route?

3. Similar to data quality, will OpenMetadata enforce any kind of responsibility onto specific users when data quality drops below a certain standard?  One of the biggest concepts in the Data Mesh world is [domain-oriented ownership](https://martinfowler.com/articles/data-mesh-principles.html#DomainOwnership), wherein the users who have the best domain knowledge of a certain section of data have a responsibility provide that data into the mesh and ensure its quality for all users within the mesh.  Is something like this in scope for OpenMetadata?

**Comment by soorisoft** [comment]
I have another follow up question for #3 from above list, How to control accessibility & visibility of data services between users or teams? As an example of Engineering vs HR teams, Engineering team doesn't need to know or see the HR department assets like internal employee pay or personal info related tables.

**Reply by harshach** [reply]
@soorisoft we have a pluggable authorizer that can limit access to datasets. It looks like you don't want anyone look at the metadata of those assets at all. Do you see a benefit of those assets to be present and discoverable by anyone? . Would like to understand the details more.

**Comment by juliarvalenti** [comment]
Following up, after writing this post I found the OpenMetadata weekly meeting VOD and my questions were covered there in some detail.

https://youtu.be/ydsS3Caejvs
